grapher/scan.py

```diff
@@ -132,11 +132,22 @@ def pkgToUnit(pkg: Dict) -> List[Unit]:
     if len(test_files) == 0:
         return [unit]
 
+    # Collect files as candidates of modules.
+    modules = []
```

Comment:
`setup.py` file only define ones the library wants to expose, but test files are using ones that are not defined in the `setup.py`'s `pkg['modules'] and pkg['py_modules'] and pkg['packages']`, thus the link would fail.